pkgs/development/tools/xcbuild/developer.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, platform, toolchain, xcbuild, writeText }:
+
+stdenv.mkDerivation {
+  name = "Xcode.app";
```

Comment:
Seems like a bit of an odd name for the derivation! I'm not sure what this is doing, actually. It's just tying together the `xcbuild` specs with the toolchain spec and so on? Given that you're `cp`ing from `xcbuild`, could it just be a separate output on the original `xcbuild` derivation?